src/ManageCourses.Api/Model/UcasInstitutionEnrichmentGetModel.cs

```diff
@@ -9,8 +10,11 @@ public UcasInstitutionEnrichmentGetModel()
             EnrichmentModel = new InstitutionEnrichmentModel();
         }
         public InstitutionEnrichmentModel EnrichmentModel { get; set; }
-
         public DateTime? CreatedTimestampUtc { get; set; }
         public DateTime? UpdatedTimestampUtc { get; set; }
+        public DateTime? LastPublishedTimestampUtc { get; set; }
```

Comment:
We can infer last published with `.Where(x => x.Status == published).OrderBy(x => x.Updated).First()`, do we need to duplicate this data?

Reply:
No we need to keep these separate. The last published date can be null for a new draft record even though the updated timestamp is set on each subsequent. Also a new draft needs to have this field set only if a previous draft has been published for the institution.